integration/cri-o/cri-o.sh

```diff
@@ -62,13 +69,25 @@ IFS=$OLD_IFS
 
 # run CRI-O tests using devicemapper on ubuntu
 if [ "$ID" == "ubuntu" ]; then
+	if [ ! -b "${LVM_DEVICE}" ]; then
+		info "Creating a loop device to use it as LVM device"
+		# create a loop device and use it as lvm device
+		trap cleanup EXIT
+		img_file=devmap.img
+		dd if=/dev/zero of=${img_file} count=10 bs=50M
+		printf "g\nn\n\n\nw\n" | fdisk ${img_file}
+		loop_device="$(sudo losetup --show -P -f ${img_file})"
+		sudo mkfs.ext4 "${loop_device}"
+		LVM_DEVICE="${loop_device}"
+	fi
+
 	# Block device attached to the VM where we run the CI
 	# If the block device has a partition, cri-o will not be able to use it.
 	export LVM_DEVICE
 	if sudo fdisk -l "$LVM_DEVICE" | grep "${LVM_DEVICE}[1-9]"; then
 		die "detected partitions on block device: ${LVM_DEVICE}. Will not continue"
 	fi
-	export STORAGE_OPTIONS="$DM_STORAGE_OPTIONS"
+	export STORAGE_OPTIONS="$DM_STORAGE_OPTIONS --storage-opt dm.directlvm_device=${LVM_DEVICE}"
```

Comment:
Probably not part of the same commit but this should no be part of the `if statement where it checks LVM_DEVICE `

Reply:
no, because the `LVM_DEVICE` can be overwritten or it can be the default device (`/dev/vdb`)